<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_UN">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PW">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>861</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_role">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>guest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>图书查询</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_year1_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_author_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_price2_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_year_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_between2_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_press_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_between1_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_adde_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_category_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_book_select_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_year2_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_title_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_author_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_price1_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_category_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>类别</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_press_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <zorder>lineEdit_year1_2</zorder>
     <zorder>label_author_2</zorder>
     <zorder>lineEdit_price2_2</zorder>
     <zorder>label_year_2</zorder>
     <zorder>label_between2_2</zorder>
     <zorder>label_press_2</zorder>
     <zorder>comboBox_type_2</zorder>
     <zorder>label_between1_2</zorder>
     <zorder>label_title_2</zorder>
     <zorder>comboBox_adde_2</zorder>
     <zorder>lineEdit_category_2</zorder>
     <zorder>pushButton_book_select_2</zorder>
     <zorder>lineEdit_year2_2</zorder>
     <zorder>label_price_2</zorder>
     <zorder>lineEdit_title_2</zorder>
     <zorder>lineEdit_author_2</zorder>
     <zorder>lineEdit_price1_2</zorder>
     <zorder>label_category_2</zorder>
     <zorder>lineEdit_press_2</zorder>
     <zorder>tabWidget</zorder>
     <zorder>tabWidget</zorder>
     <zorder>tabWidget</zorder>
     <zorder>tabWidget</zorder>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>231</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图书查询&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_select">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;xx&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>图书查询</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_price1">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_category">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_title">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_price2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_press">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_press">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_year">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_author">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_author">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_between2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_between1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_year2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_category">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>类别</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_year1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_book_select">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_adde">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_insert">
     <attribute name="title">
      <string>图书入库</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>561</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openfile">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_insert">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>入库</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_borrow">
     <attribute name="title">
      <string>借书</string>
     </attribute>
     <widget class="QLabel" name="label_borrow_cno">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>借书证卡号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_borrow_bno">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_borrow_cno">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_borrow_bno">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_borrow">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_return">
     <attribute name="title">
      <string>还书</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_return_bno">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_return_bno">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_return_cno">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_return_cno">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>借书证卡号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_return">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_card">
     <attribute name="title">
      <string>借书证管理</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_card_cno">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_card_cno">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>借书证卡号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_deletecard">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addcard">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>tabWidget_2</zorder>
   <zorder>lineEdit_UN</zorder>
   <zorder>lineEdit_PW</zorder>
   <zorder>pushButton_login</zorder>
   <zorder>tableView</zorder>
   <zorder>label_role</zorder>
   <zorder>pushButton_quit</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
